lunes_cms/cms/models/document_image.py

```diff
@@ -32,7 +32,7 @@ def image_tag(self):
         :rtype: str
         """
         if self.image and self.image.storage.exists(self.image.name):
-            if ".png" in self.image.name or ".jpg" in self.image.name:
+            if ".png" in self.image.name or ".jpg" in self.image.name.lower():
```

Comment:
```suggestion
            if any(self.image.name.lower().endswith(ext) for ext in [".jpg", ".png"]):
```
This does not work as expected, because it only covers the `.JPG` check and images with a `.PNG` extension are still not displayed correctly.
Sure, just changing it to `if ".png" in self.image.name.lower() or ".jpg" in self.image.name.lower():` would work as well, but I think it's a good opportunity to make the code more pythonic :wink: